### PR TITLE
sankalparorame.md

### PR DESCRIPTION
**Pull request by sankalparorame**
## Describe your project in 1-3 sentences
I am going to build a medical device that combines a number of devices in form of a compact and easy to use neckbandand also costs far less than all of them combined. It is aimed at overcoming the problem of delay in diagnosis and treatment of Cardiovascular Diseases (CVDs) due to lack of equipment, specially in rural areas in India. The device combines ECG, EEG and EKG modules alongside being a defibrillator for people undergoing cardiac arrest. It also comprises of a peizo modem that shall help in ascerating blood flow related issues including congestions.

## Hack Club Slack handle
@sankalparora.me

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! we're back on tuesday and will start reviewing PRs in the order that they came on. 

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: from a first glance of your project, this looks awesome and i'm wondering: are you referencing any sort of guides for this? does your budget include shipping & taxes?)_

our Jan 15 11:59pm ET deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

File: sankalparorame.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: "@sankalparorame"
+project: "TheMediBand"
+---
+
+# Project Name
```
Comment (exu3) [inline sankalparorame.md:6]: Could you update this title to be the name of your project instead of "project name"?

File: sankalparorame.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+TheMediBand
+
+## Summary
+I am going to build a medical device that combines a number of devices in form of a compact and easy to use neckbandand also costs far less than all of them combined. It is aimed at overcoming the problem of delay in diagnosis and treatment of Cardiovascular Diseases (CVDs) due to lack of equipment, specially in rural areas in India. The device combines ECG, EEG and EKG modules alongside being a defibrillator for people undergoing cardiac arrest. It also comprises of a peizo modem that shall help in ascerating blood flow related issues including congestions.
+
+## Plan
+Firstly, shall connect all sensors to Pi and program them to make them functioning and cast data to app. Later once the aimed efficiency is achieved though trial error and program refinement, I shall design a neckband case to enclose all the sensors within it to match the design and finally work on PC Client and mobile app and their connectivity.
```
Comment (exu3) [inline sankalparorame.md:13]: Do you have a Pi to use for this?

I noticed some items in your budget that I'm not sure how they are used in your project. Could you describe how you will use those items here? In particular, the Arduino. 

Also what programming languages or frameworks do you plan to use to program this?

How do you plan to design the case and what will it be made of?

**Review by exu3 — CHANGES_REQUESTED** [review]
Sounds cool! I left some comments below. Let me know if you have any questions.

File: sankalparorame.md (hunk 2 above)
Comment (exu3) [reply sankalparorame.md:12]: @sankalparorame: did you get a chance to address these questions? I saw that you requested another review but not sure if I missed something!

**Review by exu3 — APPROVED** [review]
Ok, this looks good! One more thing: could you update the PR description to include your Slack username and project description so the @hackclub/bank-ops team can reach out accordingly with next steps?

**Comment by sankalparorame** [comment]
Sure, I did the the required updation.. :)

**Review by MelSmith104 — COMMENTED** [review]
hi @sankalparorame! this project sounds very interesting and as though it would solve a very real issue going on. i'd love to hear more about this! dm'ing you on slack right now to chat about next steps :)

**Comment by daisybanks** [comment]
Closing pull request due to not being able to confirm status as high schooler.

**Pull request closed without merging**